### ISSUE TITLE
Brush Editor: don't clear input points when slider passes 0.0

### ISSUE THREAD

**Issue by briend** [mypaint/mypaint#826]
I'm sure this happens to other people.  You have a few control points, but you want to adjust the magnitude with the slider.  You grab it, and slide it a bit-- whoops, you crossed over 0.0 for a moment.  Poof!  You're points are gone.  Now you have to reload the brush or create the points again.  Even more fun is once you know about this behavior and you want to "invert" your graph and go from a positive number to a negative, you have to grab the slider and yank it really fast to be sure you don't land on 0.0 on your way to negative-land.

So, I think I the simplest thing might to just make the slider "jump over" 0.00, no matter if you are using the mouse or keyboard.  The only way you should get to 0.0 is if you press the "clear" button, I think.

**Comment by odysseywestra** [comment]
What do you know, I can actually duplicate this one! The way I was able to duplicate this what when I press `shift` once, on gnome shell it actually slows down the slider that allow me to hit exactly zero. For me that when it happens I actually hit zero. If I just zip past it to the left or right, the graph doesn't reset.

**Comment by briend** [comment]
Yep!  I realized I was starting to have this apprehensive feeling every time I dragged a slider, and I finally snapped ;-).  I knew that shift slowed down the slider while using keyboard controls, but never actually realized it worked for the mouse too.  Sweet!

**Comment by briend** [comment]
Oh, you might try Fine Speed Filter.  It has a small range from -0.20 to +0.20 so you are very likely to hit 0.00 even yanking the slider back and forth.

**Comment by achadwick** [comment]
I think based on this and #840, we're doing the right thing at zero to be honest. I can't think of a good way to scale the Y values back up after they've all been zeroed.

It'd be interesting to know where this behaviour is coming from, though :thinking:

**Comment by briend** [comment]
How do you feel about skipping 0.0? Maybe 0.0 could be mapped to .001?
